SharedCryptoPkg/Driver/CommonSharedCryptoShaOnly.c

```diff
@@ -0,0 +1,162 @@
+
+/** @file
+  This module is consumed by the reduced version of PEI
```

Comment:
the file name is more decriptive but still a good decription would be helpful and would be great to explain why this version is important or used.